src/applications/vaos/components/calendar/CalendarWidget.jsx

```diff
@@ -44,30 +44,16 @@ function getFirstDayOfMonth(momentDate) {
  * Gets the maximum month based on inputs
  *
  * @param {string} maxDate YYYY-DD-MM
- * @param {string} startMonth YYYY-MM
- * @returns {string|Moment} YYYY-MM
+ * @returns {string} YYYY-MM
  */
-export function getMaxMonth(maxDate, startMonth) {
+export function getMaxMonth(maxDate) {
   const defaultMaxMonth = moment()
     .add(DEFAULT_MAX_DAYS_AHEAD, 'days')
     .format('YYYY-MM');
+  const maxMonth = moment(maxDate).startOf('month');
 
-  // If provided start month is beyond our default, set that month as max month
-  // This is needed in the case of direct schedule if the user selects a date
-  // beyond the max date
-  if (startMonth && startMonth > defaultMaxMonth) {
```

Comment:
I'm not sure why this logic exists. The comment doesn't make sense and I don't remember setting this up at all.